### DISCUSSION TITLE
Can't find"mongo"

### DISCUSSION THREAD

**Discussion by Gandalf47** [naturalcrit/homebrewery#3281] [Q&A]
I am attempting to install Homebrewery on my device, but the program does not contain the "mongo" file referenced in the instructions for installing mongo. I have attached a screenshot.
![mongoNoIdea](https://github.com/naturalcrit/homebrewery/assets/156142567/4797dfaf-81d2-44e0-b00c-91b6f7e468db)

**Comment by calculuschild** [comment]
> I am attempting to install Homebrewery on my device, but the program does not contain the "mongo" file referenced in the instructions for installing mongo. I have attached a screenshot.
> ![mongoNoIdea](https://github.com/naturalcrit/homebrewery/assets/156142567/4797dfaf-81d2-44e0-b00c-91b6f7e468db)
> 

Which step are you stuck on specifically? I don't think we reference a file called "mongo", but you do have to type `mongo` on a command prompt or terminal on step 13.

**Comment by Gandalf47** [comment]
Right, so where should I type that?

**Reply by calculuschild** [reply]
I suggest going back to step 6 of installing mongo. That seems to be where you are getting lost:

> 6. **Open a command prompt or other terminal** and navigate to your MongoDB install folder (C:\Program Files\Mongo\Server\6.0\bin).

You should be doing most of these steps inside a command prompt.

**Reply by Gandalf47** [reply]
![mongoWutDu](https://github.com/naturalcrit/homebrewery/assets/156142567/aba2d5ea-ec17-4943-a651-3e1448570fcf)
It isn't responding.

**Reply by calculuschild** [reply]
I can see from your screenshot that you haven't completed step 6. Specifically, navigating with the terminal into the Mongo folder:

> 6. Open a command prompt or other terminal and **navigate to your MongoDB install folder (C:\Program Files\Mongo\Server\6.0\bin).**

You can find examples of how to do this from a quick online search: https://www.wikihow.com/Change-Directories-in-Command-Prompt

Also, make sure you have installed Mongo Shell, as in Step 3.

**Reply by Gandalf47** [reply]
![mongoWutdu2](https://github.com/naturalcrit/homebrewery/assets/156142567/c1f467bf-bfec-4b46-a2a7-29afe8ec6574)
It still isn't responding. Thanks for all your help by the way!
